Adds documentation for tiered caching stats API (#7158)
opensearch-project/documentation-website: _search-plugins/caching/tiered-cache.md

:--- | :--- | :--- | :---
`indices.request.cache.tiered_spillover.disk.store.policies.took_time.threshold` | Time unit | `10ms` | A policy used to determine whether to cache a query into a disk cache based on its took time. This is a dynamic setting. Optional.
`indices.request.cache.tiered_spillover.disk.store.enabled` | Boolean | `True` | Enables or disables the disk cache dynamically within a tiered spillover cache. Note: After disabling a disk cache, entries are not removed automatically and requires the cache to be manually cleared. Optional.

> ## Getting stats for the `tiered_spillover` store 

Reviewer: "stats" => "statistics"?

> ## Getting stats for the `tiered_spillover` store 

Reviewer 2: ```suggestion
## Getting statistics for the `tiered_spillover` store 
```

> To assess the impact of using the tiered spillover cache, you may want to see cache stats for each tier separately. To do this, use the API 

Reviewer 2: ```suggestion
To assess the impact of using the tiered spillover cache, use the [Node Stats API]({{site.url}}{{site.baseurl}}/api-reference/nodes-apis/nodes-stats/#caches), as shown in the following example: 

```

```json
GET /_nodes/stats/caches/request_cache?level=tier
```

> See the [nodes stats API page]({{site.url}}{{site.baseurl}}/api-reference/nodes-apis/nodes-stats/#caches) for more information. 

Reviewer 2: ```suggestion

```
